api/src/certification/enrolment/domain/usecases/get-enrolled-candidates-in-session.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @typedef {import("./index.js").EnrolledCandidateRepository} EnrolledCandidateRepository
+ */
+
+/**
+ * @function
+ * @param {Object} params
+ * @param {EnrolledCandidateRepository} params.enrolledCandidateRepository
+ * @returns [{EnrolledCandidate}]
```

Comment:
Ce n'est pas la bonne ecriture JSDoc pour ce tag : https://jsdoc.app/tags-returns

```suggestion
 * @returns {Array<EnrolledCandidate>}
```